best-practices/property-guidelines.ftd

```diff
@@ -0,0 +1,217 @@
+-- import: fastn.io/utils
+-- import: fifthtry.github.io/bling/note
+
+-- my-ds.page: Property Guidelines
+
+-- utils.code-display: `horizontal-not-left-right`: Use `horizontal` property
+id: horizontal-not-left-right
+
+When positioning elements on a web page, it is common to use the `left` and
+`right` properties to specify their horizontal placement. However, if both
+values are the same, it is more efficient to use the `horizontal` property
+instead.
+
+The `horizontal` property is a shorthand for specifying both the `left` and 
+`right` properties with a single value.
+
+-- ds.code: Not recommended
+lang: ftd
+
+\-- ftd.text: Hello World
+padding-left.px: 20 			;; <hl>
+padding-right.px: 20 			;; <hl>
+
+\;; or
+
+\-- ftd.text: Hello World
+margin-left.px: 20 				;; <hl>
+margin-right.px: 20 			;; <hl>
+
+-- ds.code: Recommended
+lang: ftd
+
+\-- ftd.text: Hello World
+padding-horizontal.px: 20 		;; <hl>
+
+
+\;; or
+
+\-- ftd.text: Hello World
+margin-horizontal.px: 20 		;; <hl>
+
+-- end: utils.code-display
+
+
+
+
+
+
+-- utils.code-display: `vertical-not-top-bottom`: Use `vertical` property
+id: vertical-not-top-bottom
+
+When positioning elements on a web page, it is common to use the `top` and
+`bottom` properties to specify their horizontal placement. However, if both
+values are the same, it is more efficient to use the `horizontal` property
+instead.
+
+The `horizontal` property is a shorthand for specifying both the `top` and 
+`bottom` properties with a single value.
+
+-- ds.code: Not recommended
+lang: ftd
+
+\-- ftd.text: Hello World
+padding-top.px: 20 				;; <hl>
+padding-bottom.px: 20 			;; <hl>
+
+\;; or
+
+\-- ftd.text: Hello World
+margin-top.px: 20 				;; <hl>
+margin-bottom.px: 20 			;; <hl>
+
+-- ds.code: Recommended
+lang: ftd
+
+\-- ftd.text: Hello World
+padding-vertical.px: 20 		;; <hl>
+
+\;; or
+
+\-- ftd.text: Hello World
+margin-vertical.px: 20 			;; <hl>
+
+-- end: utils.code-display
+
+
+
+
+
+
+-- utils.code-display: `optimize-container-props`: Applying properties to container with consistent child values
```

Comment:
@gargajit I have merged this PR, but here you have violated your `80-char` guideline. You can specify caption using:

```ftd
-- utils.code-display:
id: optimize-container-props

-- utils.code-display.caption: 

`optimize-container-props`: Applying properties to container with consistent 
child values

-- utils.code-display.body:
```

Chat with @Arpita-Jaiswal sometimes to do this properly, and also write a guideline about it.